docs/docs/data-migration.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Data migration
+---
+
+# Data migration
+
+Marathon can store its data either in ZooKeeper or in-memory. If data is stored in ZooKeeper, your data will be migrated when you upgrade Marathon.
```

Comment:
I wouldn't even talk about in-memory store. The backend is only for testing purposes and I would never encourage a user to actually use it.

Reply:
Sure. I will remove it.